msal/tests/Test.MSAL.NET.UIAutomation/CoreMobileMSALTests.cs

```diff
@@ -31,6 +31,8 @@ public static void AcquireTokenTest(ITestController controller)
             //Clear Cache
             controller.Tap("clearCache");
 
+            controller.Tap("Log");
```

Comment:
>controller.Tap("Log"); [](start = 12, length = 22)

is this on purpose? reason?

Reply:
just a test. Removed

Reply:
I still see it

---
In reply to: [216706120](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1235#discussion_r216706120) [](ancestors = 216706120)

Reply:
still around afaiks?

---
In reply to: [217874814](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1235#discussion_r217874814) [](ancestors = 217874814,216706120)